doc/source/reference/ufuncs.rst

```diff
@@ -322,7 +333,8 @@ advanced usage and will not typically be used.
 
     Accepts a boolean array which is broadcast together with the operands.
     Values of True indicate to calculate the ufunc at that position, values
-    of False indicate to leave the value in the output alone.
+    of False indicate to leave the value in the output alone. This argument
+    cannot be used for generalized ufuncs (as these take non-scalar input).
```

Comment:
Don't need either parenthesis or comma here when "as" is used with the meaning of "because".

Reply:
Don't need <- should omit.